.codespellrc

```diff
@@ -0,0 +1,5 @@
+[codespell]
+skip = *.pulsar.go,*.pb.go,*.pb.gw.go,*.cosmos_orm.go,*.json,*.git,*.js,crypto/keys,fuzz,*.h,proto/tendermint,*.bin,*.sum,*.mod,*_test.go
```

Comment:
i notice that here we skip Go test files but your PR in the relayer didn't, is this on purpose?

Reply:
yes, many test files use things like userA, step, and other words which are commonly flagged as incorrectly spelled. With the different cases it messes with codespell and is kind of a pain. so just ignoring for now